submissions/BogdanGapon/HTML & CSS Practice/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="header-nav">
+          <ul class="header-nav__list">
+            <li class="header-nav__item"><a href="" class="header-nav__link">Mail</a></li>
+            <li class="header-nav__item">
+              <a href="" class="header-nav__link" target="_blank">Images</a>
+            </li>
+            <li class="header-nav__item popup-item">
+              <label class="custom-checkbox" for="check"></label>
+              <input class="checkbox visually-hidden" type="checkbox" id="check" />
+              <span class="span"></span>
+              <div class="popup-wrapper">
+                <ul class="popup-list">
+                  <li class="popup-list__item">
+                    <a href=""
+                      ><img
+                        src="./img/apps.png"
+                        alt="Colorful circle"
+                        width="45"
+                        height="45"
+                      />Colorful circle</a
+                    >
+                  </li>
+                  <li class="popup-list__item">
+                    <a href=""
```

Comment:
This type of line breaking seems strange. Isn't it better readable like this:
```html
<a href="">
    <img src....>
    Blog
</a>
```

Reply:
Fix it everywhere, of course.

Reply:
> Fix it everywhere, of course.
Pls,can you explain why I can`t to form this tags in ul > li ( I united this elements because thease is elemnts sames -- I mean I always doing like this i unite sames elements in ul and its better for me. Why I can`t do it like this ?